### ISSUE TITLE
Texture.draw should take into account the current scale attribute

### ISSUE THREAD

**Issue by acbart** [pvcraven/arcade#516]
## Bug Report
`load_texture` with a `scale` parameter doesn't seem to scale the texture.

### Actual behavior:
The .width or .height value of the texture are unchanged after you load a texture with a scale parameter.

### Expected behavior:
I would expect the size when drawn to be different, and the .width and .height values to be changed.

### Steps to reproduce/example code:

```python
import arcade

arcade.open_window(50, 50, "")

BIG = arcade.load_texture("ada.png", scale = 1)
SMALL = arcade.load_texture("ada.png", scale = .5)

assert BIG.width > SMALL.width
```

Causes an assertion error because they are the same values.

**Comment by pvcraven** [comment]
Thanks for the report. Usually textures are scaled through the sprite, so this appears to have been overlooked.

**Comment by pvcraven** [comment]
Fixed with version 2.2.8.
